compact/tracking/tof_endcap.xml

```diff
@@ -11,10 +11,10 @@
     <constant name="ForwardTOF_zmin"                value="ForwardTOFRegion_zmin + ForwardTOFRegion_length - ForwardTOF_length - 1*cm"/>
     <constant name="ForwardTOF_nlayers"             value="1"/>
     <constant name="ForwardTOFMod_offset"           value="1.2*cm"/>
-    <constant name="BackwardTOF_length"             value="min(BackwardTOFRegion_length, 2.*cm)"/>
+    <!-- <constant name="BackwardTOF_length"             value="min(BackwardTOFRegion_length, 2.*cm)"/>
```

Comment:
In this file we should remove commented out parameter blocks. They are still available in the git history and they are not tested in code, so they are bound to get out of date.